src/Layers/xrRender/HOM.cpp

```diff
@@ -75,6 +75,7 @@ void CHOM::Load()
         IReader* S = fs->open_chunk(1);
         const auto begin = static_cast<HOM_poly*>(S->pointer());
         const auto end   = static_cast<HOM_poly*>(S->end());
+        CL.reserve_faces(S->length());
```

Comment:
Using `S->length()` is wrong here.
There's so much pull requests from you with many mistakes, so I won't tell you why this code is wrong. Learn it yourself, please, and raise the quality of you pull requests.

Reply:
This may sound bitchy :) but wasting time and energy on reviewing continuous mistakes is not good too.